### DISCUSSION TITLE
ESD option

### DISCUSSION THREAD

**Discussion by sunzhe-96** [gmtsar/gmtsar#296] [General]
Hello,everyone,I want to call the ESD option, what should I do about that？

**Comment by Xiaohua-Eric-Xu** [comment]
You could, turn the ESD by changing `spec_div = 1` in the config file for p2p_processing.csh, with setting the mode using `spec_mode = x`.  mode = 0 for constant sum correction, mode = 1 for constant median correction, mode = 2 for non-constant correction with mapping the residual azimuth shift. For batch processing, use `preproc_batch_tops_esd.csh`

**Reply by sunzhe-96** [reply]
> You could, turn the ESD by changing `spec_div = 1` in the config file for p2p_processing.csh, with setting the mode using `spec_mode = x`. mode = 0 for constant sum correction, mode = 1 for constant median correction, mode = 2 for non-constant correction with mapping the residual azimuth shift. For batch processing, use `preproc_batch_tops_esd.csh`

Thank you, Eric, I'll try.
